src/signals/incident-management/containers/IncidentSplitContainer/styled.js

```diff
@@ -73,8 +73,34 @@ export const StyledLabel = styled(Label)`
   }
 `
 
+export const StyledMainContainer = styled.div`
+  display: grid;
+  grid-column: span 7;
+  row-gap: ${themeSpacing(8)};
+
+  @media screen and ${breakpoint('max-width', 'laptopM')} {
+    grid-column: span 10;
+  }
+
+  @media screen and ${breakpoint('max-width', 'laptop')} {
+    grid-column: span 12;
+  }
+`
+
+export const StyledButtonContainer = styled.div`
+  grid-column: span 7;
+`
+
 export const StyledFieldset = styled.fieldset`
+  background-color: ${themeColor('tint', 'level3')};
   scroll-margin-top: ${themeSpacing(15)};
+  grid-column: span 12;
+  padding: ${themeSpacing(5)};
+  margin-inline: -${themeSpacing(5)};
+  display: grid;
+  column-gap: ${themeSpacing(16)};
+  row-gap: ${themeSpacing(8)};
+  grid-template-columns: 7fr 5fr;
 `
 
 export const StyledHeading = styled(Heading)`
```

Comment:
Lineheight is 30px but should b 24 px.

Reply:
Hm, I see that's in the design, but would mean a break from the Amsterdam brand style. I'll ask Linda if this is intentional

Reply:
Checked with Linda, it was unintentional